### PR TITLE
Core/ts version task 502 stress script

### PR DESCRIPTION
**Pull request by WilliamHawkingStephen**
## What does this MR do?

- add scripts for stress test 
- execute able from terminal 
- update readme 

## Check list  

- [x] All Merge Request pipelines should be passed.
- [x] Make sure all the changes are necessary and correct.
- [x] Make sure no needed change missing (all necessary changes are made).
- [x] Merge Request should contains only changes for one purpose (one feature or bug or refactoring).
- [ ] All necessary unit tests are written.
- [x] Critical changes should be tested locally (or on staging).

### CODE REVIEW

File: ts-version/src/test/scripts/.env.script.example [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ALICE_WALLET_PRIV_KEY=c53a44d38401569c997286be0f6d2ab22e8750e953bd2e1db93e4dfe4d648940
+BOB_WALLET_PRIV_KEY=7912fe31eba43a9227a2bbeb3fb7caf4c79c91f5c174028b60321eba85681e9c
+BACKEND_HTTP_URL=http://localhost:3004
```
Comment (GreatGaryCooper) [inline ts-version/src/test/scripts/.env.script.example:3]: EOL error, please add a new line after the last line.

File: ts-version/src/test/scripts/.env.script [hunk 2]
```diff
@@ -0,0 +1,3 @@
+ALICE_WALLET_PRIV_KEY=c53a44d38401569c997286be0f6d2ab22e8750e953bd2e1db93e4dfe4d648940
+BOB_WALLET_PRIV_KEY=7912fe31eba43a9227a2bbeb3fb7caf4c79c91f5c174028b60321eba85681e9c
+BACKEND_HTTP_URL=http://localhost:3004
```
Comment (GreatGaryCooper) [inline ts-version/src/test/scripts/.env.script:3]: EOL error, please add a new line after the last line.

**Review by GreatGaryCooper — APPROVED** [review]
LGTM

File: ts-version/src/test/scripts/bot.js [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+    this.networks = networks;
+    this.chainId = chainId;
+    this.privateKey = privateKey;
+  }
+
+  signIn = async () => {
+    this.ethersProvider = getDefaultProvider("goerli");
+    this.ethWallet = new Wallet(this.privateKey, this.ethersProvider);
+
+    let message = "Login to Dexpresso";
```
Comment (hypersolo) [inline ts-version/src/test/scripts/bot.js:96]: .env is a better place for message!

File: ts-version/src/test/scripts/bot.js [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+
+    axios
+      .post(`${this.BACKEND_https_URL}/login`, {
+        network: "ethereum_goerli",
+        user_address: this.ethWallet.address,
+        signature: flatSig,
+        user_data: false,
+      })
+      .then((res) => {
+        console.log(res);
```
Comment (hypersolo) [inline ts-version/src/test/scripts/bot.js:108]: one of these two logs is enough!

**Review by GreatGaryCooper — APPROVED** [review]

**Pull request merged**